GitHub_Learning/Feature/Onboarding/Application/PageController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  PageController.swift
+//  GitHub_Learning
+//
+//  Created by Amin Ashhab on 06/11/2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import Foundation
+import CleanCore
+import CleanPlatform
+
+protocol PagesController: BaseController {
```

Comment:
IMO (in my opinion) nazval bych to spis `OnboardingController`. Pages je moc obecny.